src/app/components/home.component.ts

```diff
@@ -1280,6 +1288,11 @@ export class HomeComponent implements OnInit, AfterViewInit {
         }, 300);
       }
     }
+    if (!fromIpc) {
```

Comment:
This should include `codeRunningOnMac` and probably a `touchBarEnabled` variable - see #312